lib/puppet/provider/scheduled_task/taskscheduler_api2.rb

```diff
@@ -105,8 +105,8 @@ def trigger
       when Win32::TaskScheduler::TASK_TIME_TRIGGER_ONCE
         puppet_trigger['schedule'] = 'once'
       end
-      puppet_trigger['start_date'] = self.class.normalized_date(trigger['start_year'], trigger['start_month'], trigger['start_day'])
-      puppet_trigger['start_time'] = self.class.normalized_time(trigger['start_hour'], trigger['start_minute'])
+      puppet_trigger['start_date'] = PuppetX::PuppetLabs::ScheduledTask::Trigger::V1.normalized_date(trigger['start_year'], trigger['start_month'], trigger['start_day'])
```

Comment:
Typo in commit message ((MODULES-6895) Move normalized helpers to V1 class): "codd" -> "code"